activesupport/test/actionable_error_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+require "active_support/actionable_error"
+
+class ActionableErrorTest < ActiveSupport::TestCase
+  class NonActionableError < StandardError
+  end
+
+  class DispatchableError < StandardError
+    include ActiveSupport::ActionableError
+
+    class_attribute :flip1, default: false
+    class_attribute :flip2, default: false
+
+    action "Flip 1" do
+      self.flip1 = true
+    end
+
+    action "Flip 2" do
+      self.flip2 = true
+    end
+  end
+
+  test "can get all action of an actionable error" do
+    assert_raises ActiveSupport::ActionableError::NonActionable do
+      ActiveSupport::ActionableError.actions(NonActionableError)
+    end
+  end
+
+  test "cannot get actions from non-actionable errors" do
```

Comment:
I guess you swapped this description with the one above.

Reply:
Yup, thanks for the catch!